reana/cli.py

```diff
@@ -1524,15 +1526,12 @@ def setup_environment(server_hostname, insecure_url):  # noqa: D301
         export_lines.append(component_export_line.format(
             env_var_name='REANA_SERVER_URL',
             env_var_value=server_hostname or get_external_url(insecure_url)))
-
-        get_admin_token_sql_query_cmd = [
-            'psql', '-U', 'reana', 'reana', '-c',
-            'SELECT token FROM user_token']
-        sql_query_result = exec_into_component(
-            get_prefixed_component_name('db'),
-            get_admin_token_sql_query_cmd)
-        # We get the token from the SQL query result
-        admin_access_token = sql_query_result.splitlines()[2].strip()
+        get_access_token_cmd = \
+            'kubectl get secret -o json reana-dev-admin-access-token'
```

Comment:
I think it should be `get_prefixed_component_name('admin-access-token')` or similar.

Reply:
Absolutely, thanks a lot for spotting it :)